test/scalability/placement_test.go

```diff
@@ -18,19 +19,21 @@ import (
 )
 
 const (
-	clusterSetLabel = "cluster.open-cluster-management.io/clusterset"
-	placementLabel  = "cluster.open-cluster-management.io/placement"
+	clusterSetLabel   = "cluster.open-cluster-management.io/clusterset"
+	placementLabel    = "cluster.open-cluster-management.io/placement"
+	placementSetLabel = "cluster.open-cluster-management.io/placementset"
 )
 
 var _ = ginkgo.Describe("Placement scalability test", func() {
 	var namespace string
-	var placementName string
-	var clusterSet1Name string
+	var placementSetName string
+	var clusterSetName string
 	var suffix string
+	var sampleCount = 10
 	var err error
 
 	assertPlacementDecisionCreated := func(placement *clusterapiv1alpha1.Placement) error {
-		ginkgo.By("Check if placementdecision is created")
+		//ginkgo.By("Check if placementdecision is created")
```

Comment:
This function will be called by `assertCreatingPlacement` which will be called once a placement is created, as there will hundres of placements in the new test cases, this message will be printed many times. Comment it out here to avoid the flushed messages.

Reply:
let's just remove this line.